Frontend/src/utils/utils.js

```diff
@@ -8,6 +8,36 @@ function saveParseTimestampToString(millis) {
     )}.${date.getFullYear()}`;
 }
 
+
+
+function saveParseTimestampToHumanReadableString(millis) {
+  const date = new Date(millis);
+  let dayDiff = daysBetween(date, new Date().getTime());
```

Comment:
```suggestion
  let dayDiff = daysBetween(date, millisAtStartOfToday());
```
All timestamps are stored at 00:00 UTC+0, so we should only compare them against timestamps at 00:00. Otherwise daysBetween returns -1 if `new Date().getTime()` > 12:00 UTC+0.